### ISSUE TITLE
[Feature Request] Support for Less

### ISSUE THREAD

**Issue by GNRSN** [storybookjs/addon-styling#17]
Hey guys, great work on streamlining process of adding css pre-processing. 

In our current storybook 6 implementation the only manual edit to webpack-config we have to do is adding support for [less](https://github.com/less/less.js) through [less-loader](https://github.com/webpack-contrib/less-loader). 

The process is similar to adding sass support to webpack so it would be great if addon-styling could provide storybook "out of the box"-support for less as well.

**Comment by GNRSN** [comment]
Hi @Integrayshaun, a lot have happened in the time since I posted this. 

As part of our Storybook 7 migration we decided to start migrating our code-base to scss as we deemed the community support with tooling around scss to be superior. I spent my easter downtime migrating our legacy-frontends from an early boostrap 3 version that used less to the alternate sass-package to complete the transition 😛. 

With that said, I/we don't really have any investment in this Issue. Feel free to close it unless you want to proceed on your own initiative.
